.travis.yml

```diff
@@ -24,6 +25,10 @@ env:
   - secure: TR1UN2r3beDtIF+VJpLF2ocTv/uxuOKyVWWhzMLre0ZrrBaIP1sLZV7Z4S/km5M92EfPgGW87BogdWE/R+kTTRiPTCbFB/U/3jFKxEZRXKST66YK5JMwsYqb17UtZtFdqEtO9GGbAVzXwpZfMMoXvlKNrors2W32xBm2uIkOSpI=
   - secure: Q8RH65NClRBryxfvlwHQjeR4wGs+GXeUBRIBd1kspAM7Uv+5K3iP+q/TPCIrTL/OpJ5tyCYfeq5hKIFFwrY3JikNXnyHICOjXPE6zSLUm8E8NHaP/orPNWjze2x4yDSRCifqzr2ZiXVq9sxlZfNbZ9eyJxpPFTYDSpuN9T5UEE0=
 
+matrix:
+  allow_failures:
+    - env: NODE_VERSION="lts/*" TEST_SUITE=performance
```

Comment:
How does `allow_failures` translate to Slack and Github? If the perf tests fail but everything else passes does it look green on both of those platforms?

Reply:
Green everywhere, afiact